lib/src/nakama_client/nakama_grpc_client.dart

```diff
@@ -368,4 +367,28 @@ class NakamaGrpcClient extends NakamaBaseClient {
       options: _getSessionCallOptions(session),
     );
   }
+
+  @override
+  Future<LeaderboardRecordList> listLeaderboardRecords({
+    required model.Session session,
+    required String leaderboardId,
+    List<String>? ownerIds,
+    int limit = 20,
+    String? cursor,
+    String? expiry,
```

Comment:
Imho this should be a `DateTime` instead of a string so that users understand what value to pass here. The `expiry` field is seconds since epoch, so you can then convert as follows:

```dart
expiry.millisecondsSinceEpoch / 1000
```